package.json

```diff
@@ -78,20 +92,5 @@
     },
     "dependencies": {
         "semver": "^5.3.0"
-    },
-    "configuration": {
-        "type": "object",
-        "title": "VSCode vale",
-        "properties": {
-            "vscode-vale.path": {
-                "type": [
-                    "string",
-                    "null"
-                ],
-                "default": null,
-                "description": "Specifies the path to the vale executable, useful if you don't want to use the global binary.",
-                "scope": "machine"
```

Comment:
PS: I wasn't sure about the [scope](https://code.visualstudio.com/api/references/contribution-points) value, so I removed it (the default would be `window`).